### ISSUE TITLE
Linux: incorrect directory for make install

### ISSUE THREAD

**Issue by atsb** [andrei-drexler/ironwail#84]
Having IronWail be installed in (by default) /usr/local/games/quake does not make much sense as it cannot be executed via the terminal and does not appear in the start menus of pretty much every distro.

Having a sane default of /usr/local/bin would be much better.

**Comment by imavlr** [comment]
Uh, steam installs to /usr/games
/usr/local/games exists as well by default in debian

**Comment by 4LT** [comment]
I don't believe there's a one-size-fits-all solution to this, different distros have different conventions. Personally, I like to install to a location I don't need to use `sudo` for (`~/bin` although I'd have used `~/.local/bin` if I had been aware of it). Sometimes I'll even create a symlink to the repo so I don't have to worry about the installation step.

**Comment by colby-boucher** [comment]
I'm having something a little different happen along the same lines in Fedora, the two commands that are supposed to be executing literally don't do anything somehow. The only thing that ends up in /usr/local/games/quae is quake.pak even though cp ironwail /usr/local/games/quake and cp ironwail.pak /usr/local/games/quake come up like they should. I need to copy them over manually.
